### ISSUE TITLE
Magic Inverter Type Strings

### ISSUE THREAD

**Issue by h0bb3** [srcfl/srcful-gateway#138]
The inverter type string is crucial for saving the inverter and for the backend.

Currently, this is undocumented and patched in Bootstrap and HarvestTransport to accommodate different casing.

Backend id should be a field in the harvest configs, inverter should have a method for getting this that should be used instead of type.

the get method in the profiles should be more forgiving and probably ignore casing and also allow for backend id to be used as identifier.

In short having the casing in e.g. the boostrap file cause problems all the way to the backend is not a good thing.

Add code comments for the parts that must not be changed if consequences are not known....

Possibly add test cases that check that the correct things are loaded and also sent to the backend.

**Comment by h0bb3** [comment]
Is this being worked on? I would really like to avoid having all eGWs go offline when releasing... @davmoz

**Comment by h0bb3** [comment]
This seems to have been fixed.
